kuma/javascript/src/header/header.jsx

```diff
@@ -153,18 +92,112 @@ type Props = {
+                    {
+                        url: `/${locale}/docs/Web/API`,
+                        label: gettext('APIs / DOM')
+                    },
+                    {
+                        url: `/${locale}/docs/Mozilla/Add-ons/WebExtensions`,
+                        label: gettext('Browser Extensions')
+                    },
+                    {
+                        url: `/${locale}/docs/Web/MathML`,
+                        label: gettext('MathML')
+                    }
+                ]
+            },
+            {
+                label: gettext('References & Guides'),
+                url: `/${locale}/docs/Learn`,
+                items: [
+                    {
+                        url: `/${locale}/docs/Learn`,
+                        label: gettext('Learn web development')
+                    },
+                    {
+                        url: `/${locale}/docs/Web/Tutorials`,
+                        label: gettext('Tutorials')
+                    },
+                    {
+                        url: `/${locale}/docs/Web/Reference`,
+                        label: gettext('References')
+                    },
+                    {
+                        url: `/${locale}/docs/Web/Guide`,
+                        label: gettext('Developer Guides')
+                    },
+                    {
+                        url: `/${locale}/docs/Web/Accessibility`,
+                        label: gettext('Accessibility')
+                    },
+                    {
+                        url: `/${locale}/docs/Games`,
+                        label: gettext('Game development')
+                    },
+                    {
+                        url: `/${locale}/docs/Web`,
+                        label: gettext('...more docs')
+                    }
+                ]
+            },
+            {
+                label: gettext('Feedback'),
+                url: `/${locale}/docs/MDN/Feedback`,
+                items: [
+                    {
+                        url: 'https://support.mozilla.org/',
+                        label: gettext('Get Firefox help'),
+                        external: true
+                    },
+                    {
+                        url: 'https://stackoverflow.com/',
+                        label: gettext('Get web development help'),
+                        external: true
+                    },
+                    {
+                        url: `/${locale}/docs/MDN/Community`,
+                        label: gettext('Join the MDN community')
+                    },
+                    {
+                        label: gettext('Report a content problem'),
+                        external: true,
+                        url: `https://github.com/mdn/sprints/issues/new?template=issue-template.md&projects=mdn/sprints/2&labels=user-report&title=${encodeURIComponent(
+                            props.document ? props.document.slug : '/' + locale
```

Comment:
This does not match with line 199. But perhaps I'm missing the point anyway. 
I.e. if you change the page, the `useMemo` is re-run anyway. So there's no memoization across pages. 

Either way, it's just odd that the input hint on line 199 talks about `locale` but the bulk of the code actually depends on `[props.document, locale]`. Right?

Reply:
There's definitely something wrong. That link isn't updating as I changed pages. It sticks to whatever page I was on the first SSR load. 

By the way, potentially a new/different but, but this is what I get when I clicked on the "Rapportera a problem with the content" (but in Swedish):
<img width="750" alt="Screen Shot 2019-06-27 at 3 57 02 PM" src="https://user-images.githubusercontent.com/26739/60296811-e3a20880-98f4-11e9-92c1-44320ec48bd3.png">
It's lacking information about the fact that I was on the Swedish page.

Reply:
You're right that useMemo() should also be based on props.document. I'll fix that.

Reply:
You're also right that the bug report form ought to include the locale. It is unrelated but I'll see if I can fix that now.